pac4j-saml/pom.xml

```diff
@@ -215,11 +225,18 @@
             <id>shib-release</id>
             <url>https://build.shibboleth.net/nexus/content/groups/public</url>
             <snapshots>
-                <enabled>false</enabled>
+                <enabled>true</enabled>
```

Comment:
This will need to be changed when the final release of opensaml 3.4 is available